README.md

```diff
@@ -152,87 +201,15 @@ The wizard will ask for 1 option -
-  - Flow actually understands `propTypes` in `createClass` calls but not ES6 class components. Here the transformation logic is identical to [how](https://github.com/facebook/flow/blob/master/src/typing/statement.ml#L3526) Flow treats `propTypes`
-  - Notice that Flow treats an optional propType as non-nullable
-    - For example, `foo: React.PropTypes.number` is valid when you pass `{}`, `{foo: null}`, or `{foo: undefined}` as props at **runtime**. However, when Flow infers type from a `createClass` call, only `{}` and `{foo: undefined}` are valid; `{foo: null}` is not. Thus the equivalent type annotation in Flow is actually `{foo?: number}`. The question mark on the left hand side indicates `{}` and `{foo: undefined}` are fine, but when `foo` is present it must be a `number`
-  - For `propTypes` fields that can't be recognized by Flow, `$FlowFixMe` will be used
-5. `React.createClass` is no longer present in React 16. So, if a `createClass` call cannot be converted to a plain class, the script will fallback to using the `create-react-class` package.
-  - Replaces `React.createClass` with `ReactCreateClass`.
-  - Adds a `require` or `import` statement for `create-react-class`. The import style is inferred from the import style of the `react` import. The default module name can be overridden with the `--create-class-module-name` option.
-  - Prunes the `react` import if there are no more references to it.
-
-#### Usage
-```bash
-npx react-codemod class <path>
-```
-
-### jscodeshift options
-
-To pass more options directly to jscodeshift, use `--jscodeshift="..."`. For example:
-```sh
-npx react-codemod --jscodeshift="--run-in-band --verbose=2"
-```
-
-See all available options [here](https://github.com/facebook/jscodeshift#usage-cli).
-
-### Recast Options
-
-Options to [recast](https://github.com/benjamn/recast)'s printer can be provided
-through jscodeshift's `printOptions` command line argument
-
-```sh
-npx react-codemod <transform> <path> --jscodeshift="--printOptions='{\"quote\":\"double\"}'"
-```
-
-#### `explicit-require=false`
-
-If you're not explicitly importing React in your files (eg: if you're loading React with a script tag), you should add `--explicit-require=false`.
-
-### Support and Contributing
+## Support and Contributing
 
-The scripts in this repository are provided in the hope that they are useful,
-but they are not officially maintained, and we generally will not fix
-community-reported issues. They are a collection of scripts that were previously
-used internally within Facebook or were contributed by the community, and we
-rely on community contributions to fix any issues discovered or make any
-improvements. If you want to contribute, you're welcome to submit a pull
-request.
+The scripts in this repository are maintained by the React team in collaboration with the [codemods.com](https://codemods.com) team.
```

Comment:
@rickhanlonii Typo: Extra 's'. should be codemod.com.

Reply:
```suggestion
The scripts in this repository are maintained by the React team in collaboration with the [codemod.com](https://codemod.com) team.
```